tests/python_package_test/test_dask.py

```diff
@@ -79,6 +79,7 @@ def _create_ranking_data(n_samples=100, output='array', chunk_size=50, **kwargs)
     X, y, g = make_ranking(n_samples=n_samples, random_state=42, **kwargs)
     rnd = np.random.RandomState(42)
     w = rnd.rand(X.shape[0]) * 0.01
+
```

Comment:
Not a linter thing, yeah just a gaffe from undoing changes from 3952. I think if I just get rid of it this and push it will go away - sorry I'm not sure if that affects git blame history if I make a change in this branch to a change made in 3952?

Reply:
nope not at all! We squash commits so you'll be good

Reply:
Resolved in be091106